src/views/forms/NewMarketForm/v7/NewMarketSelectionStep.tsx

```diff
@@ -53,6 +55,8 @@ export const NewMarketSelectionStep = ({
   const isDisconnected = onboardingState === OnboardingState.Disconnected;
   const launchableMarkets = useLaunchableMarkets();
   const stringGetter = useStringGetter();
+  const vault = useLoadedVaultDetails().data;
+  const depositApr = vault?.thirtyDayReturnPercent && vault.thirtyDayReturnPercent / 100;
```

Comment:
I don't think we need to divide by 100, it's already ready to be passed into the Output.